examples/riot/buildAll.sh

```diff
@@ -21,3 +21,8 @@ for board in "${BOARDS[@]}"; do
 			popd
 	done
 done
+
+# Build lf example
```

Comment:
@LasseRosenow Is this needed?

Reply:
I think this should be fixed when you rebase with main. This does not exist anymore :)

Reply:
I did a merge with main and ended up with this, will remove it